Kitodo-API/src/main/java/org/kitodo/api/ugh/ContentFileInterface.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.ugh;
+
+/**
+ * A ContentFile represents a file which must be accessible via the file system
+ * and contains the content of the {@code DigitalDocument}. A ContentFile
+ * belongs always to a {@code FileSet}, which provides methods to add and remove
+ * content files ({@code addFile} and {@code removeFile}. ContentFile objects
+ * are not only be part of a FileSet but must also be linked to structure
+ * elements. Therefore references to {@code DocStruct} objects exists.
+ */
+public interface ContentFileInterface {
+    /**
+     * Returns the location of the content file. The filename is always an
+     * absolute file.
+     *
+     * @return the location of the content file
+     */
+    String getLocation();
+
+    /**
+     * Sets the location of the content file. The file must at least be readable
+     * from this position.
+     *
+     * @param fileName
+     *            the location of the content file
+     * @return always {@code true}. Return value is never used.
```

Comment:
Why should this method implementation return always true if this value is never used? If so then the method should be adjusted or return value used. I would prefer the first case: adjust the method.

Reply:
This is an interface to existing UGH methods. It must not change method signatures as they are in the UGH. I think the way the UGH was implemented is wrong. But I cannot help it.

Reply:
So introducing a wrong interface is the better way then fixing this in existing code? As you know this kind of interface will longer exists as expected and then every one must implement this in a wrong way. Sorry, I could not accept this.